release.sh

```diff
@@ -1,74 +1,161 @@
 #!/usr/bin/env bash
 
 set -e -o pipefail
+shopt -s extglob
 
-if [ $# -lt 1 ]; then
+VERSION=
+NEXT_VERSION=
+BRANCH=$(git rev-parse --abbrev-ref HEAD)
+NARGS=0
+DRY_RUN=
+INSANE=0
+
+# usage [ERROR-MESSAGES...] EXIT-STATUS
+usage() {
+    while [[ $# -gt 1 ]]; do
+        echo "$1" >&2
+        shift
+    done
     echo "Usage:"
-    echo "./release.sh <VERSION_TO_RELEASE> [NEXT_VERSION]"
+    echo "./release.sh [OPTIONS] <VERSION_TO_RELEASE> [NEXT_VERSION]"
```

Comment:
I've been asking myself this very question just now. Benefit of doing so is that we get more up-to-date information on the release branch: updated URLs, updated sri hashes, updated version number. The downside is that we'd have one more commit on the branch, but actually that might be seen as a benefit, too, since otherwise it's hard to place the release in the line of development. Do we want to enforce using a second argument?